opentelemetry/proto/collector/profiles/v1development/profiles_service.proto

```diff
@@ -37,6 +37,9 @@ message ExportProfilesServiceRequest {
   // data from multiple origins typically batch the data before forwarding further and
   // in that case this array will contain multiple elements.
   repeated opentelemetry.proto.profiles.v1development.ResourceProfiles resource_profiles = 1;
+
+  // The lookup tables containing all data shared by profiles across the message being sent.
+  repeated opentelemetry.proto.profiles.v1development.ProfilesReferenceTables lookup_tables = 2;
```

Comment:
Given that this is proposed to be a pattern for all signals, can we unify the terminology?

We use _index in the field names to reference the items in the tables.

This field uses both "lookup" and "reference" terms.

This seems too many.

Reply:
How about "dictionary" here (both type and field) and _index in fields?

Reply:
It's unlikely any other signal will start doing this without breaking changes (and thefore v2 and all that entitles).
But sure, I'm open to any name proposal.

Reply:
`ProfilesDictionary dictionary` SGTM. I think we're already using `_index` everywhere we should?

Reply:
Yes, I think the `_index` / `_strindex` naming style is uniform already.

Reply:
I like the term dictionary as well.

Reply:
I have switched the term to dictionary.